delta-lake/delta-spark332db/src/main/scala/com/databricks/sql/transaction/tahoe/rapids/GpuDeltaDataSource.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022-2025, NVIDIA CORPORATION.
```

Comment:
Isn't this a new file?

```suggestion
 * Copyright (c) 2025, NVIDIA CORPORATION.
```

Reply:
It's tricky to answer that. This file is copied over from the common directory to specific delta directories. Should we go with what git thinks or should we go with what actually happened in this commit is the question I ask everytime I do this.